index.js

```diff
@@ -42,6 +42,18 @@ function servePage(path, res) {
     });
 }
 
+async function validToken(token){
```

Comment:
Function returns a teamid, not a boolean.
Name needs to reflect the fact that it returns a teamid.